MAINTAINERS

```diff
@@ -4,20 +4,5 @@
   "issues": "https://tickets.puppetlabs.com/browse/SERVER",
   "internal_list": "https://groups.google.com/a/puppet.com/forum/?hl=en#!forum/discuss-puppetserver-maintainers",
   "people": [
```

Comment:
want to just delete this file?

Reply:
I kinda just wanted to add you as schadenfreude

Reply:
i know where you are

Reply:
No you don't. I'm not at my desk

Reply:
Damnit